### PR TITLE
update examples, set v3 for provider

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Updates examples with v3 provider version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ nav_order: 1
 
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
 
+## [3.6.1] - 2022-09-05
+
```
Comment (Serpentiel) [inline CHANGELOG.md:13]: please move all these changes under `[MAJOR.MINOR.PATCH] - YYYY-MM-DD` section

Comment (Serpentiel) [inline CHANGELOG.md:13]: please move all these changes under `[MAJOR.MINOR.PATCH] - YYYY-MM-DD` section
we add version number when we're ready to release

**Review by Serpentiel — CHANGES_REQUESTED** [review]
there might be some chance that these examples won't exactly work with v3.x of our provider

also, at least one of these resources should be removed, that is the `elasticsearch` resource, and we should probably update it to use `opensearch`

we should probably discuss this a bit more before proceeding next

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Just a minor thing, can you put a space in between `>=` and the version?

**Comment by ivan-savciuc** [comment]
@byashimov can you check comments for this PR please?

**Pull request closed without merging**